### ISSUE TITLE
Alt text for figures

### ISSUE THREAD

**Issue by kfosnacht** [rubenarslan/codebook#37]
Can you automatically add alt-text to the distribution plots to meet WCAG accessibility standards? An alt-text of "Distribution of *var*" should be sufficient.

FYI, The other violations per WAVE are missing form labels in the codebook table, document language missing, and an empty table header in the missingness table.
